doc/devdocs/settings-reference.md

```diff
@@ -80,7 +80,7 @@ Example module JSON (taken from Shortcut Guide):
 ```
 produces this settings screen:
 
-![Shortcut Guide settings](../images/settings/shorcut_guide_settings.png)
+![Shortcut Guide settings](../images/settings/shortcut_guide_settings.png)

```

Comment:
This change requires to fix the filename of https://github.com/jsoref/PowerToys/blob/spelling/doc/images/settings/shorcut_guide_settings.png

Otherwise the image is not found in the readme:

![image](https://user-images.githubusercontent.com/3206696/82866314-f2d86e80-9f28-11ea-9cb9-3e9b6039ea87.png)